frontend/src/components/dropdown/Dropdown.css

```diff
@@ -0,0 +1,16 @@
+/* Dropdown radio options */
```

Comment:
Since our dropdowns look pretty different from the Ant Design ones, to change the styling we have to directly access the CSS classes on the elements we want to modify